index.html

```diff
@@ -171,6 +174,35 @@ <h1>Profesional Art Printing Data</h1>
                 <button class="see-project-button" type="button">See Project</button>
             </div>
         </div>
+        <div class="project-details-modal-background disappear">
+            <div class="project-details-modal">
+                <div class="project-modal-header">
+                    <div class="modal-head">
+                        <h3 class="modal-heading-mobile">Multi Post Stories</h3>
+                        <h3 class="modal-heading-desktop">Keeping track of hundreds  of components website</h3>
+                        <i class="dark-cross-button"></i>
+                    </div>
+                    <ul class="languages">
+                        <li><span>html</span></li>
+                        <li><span>Bootstrap</span></li>
+                        <li><span>Ruby on rails</span></li>
+                    </ul>
+                </div>
+                <div class="modal-content">
+                    <img src="images/portfolio/snapshoot-portfolio.png" alt="Portfolio snapshot image">
+                    <div class="modal-lower-half">
+                        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+                            been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                            galley of type and scrambled it to make a type specimen book. It has survived not only five
+                            centuries, but also the leap into electronic typesetting, remaining essent</p>
+                        <div class="modal-buttons">
+                            <div class="btns">See Live</div>
+                            <div class="btns">See Source</div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>

```

Comment:
- [ ] Kindly make sure to display the modal properly on mobile view. Your modal is too long and therefore overflows the device's height.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/161792289-3b0f635f-69ec-4524-aa68-f3803085cec2.png) | ![image](https://user-images.githubusercontent.com/62290545/161792617-bb9160e5-0173-46d0-9ed0-29923f7f5794.png) |

Reply:
All good now 👍🏽